docs/index.html

```diff
@@ -20,150 +21,283 @@
     <link rel="icon" href="img/favicon.ico" type="image/x-icon"/>
 
     <style>
+        #tweets-container {
```

Comment:
Passing by really :walking_man: But Curious why you have another style declaration.

Reply:
May I know why?